doc/guides/weight_normalization.rst

```diff
@@ -0,0 +1,51 @@
+Weight normalization
+====================
+
+Preliminaries
+-------------
+
+Suppose that the incoming synaptic weights of a neuron are given as :math:`w_1, w_2, \ldots, w_n`. A plasticity rule might require that the vector norm :math:`|w|` remains constant. For example, the 1-norm is used in [1]_ [2]_:
+
+.. math::
+
+   |W|_1 = \sum_i W_i
+
+Keeping this norm constant is typically done as an extra step after the main weights plasticity step (e.g. after an STDP weight update). First, the norm is computed, and second, all weights :math:`w_1, \ldots, w_n` are updated according to:
+
+.. math::
+
+   W_i &\leftarrow W_{total} \frac{W_i}{|W|_1}
+
+where :math:`w_{total}` is the target total input weight [2]_.
```

Comment:
I think it would be easier to follow if `w_{total}` was defined above, for instance: 
"... remains constant, e.g., at the target value :math:`w_{total}`."